### ISSUE TITLE
LDAP Documentation Adjustment

### ISSUE THREAD

**Issue by Jemikwa** [digitalocean/netbox#2656]
### Change Type
[x] Addition
[ ] Correction
[ ] Deprecation
[ ] Cleanup (formatting, typos, etc.)

### Proposed Changes
1. Either appending or changing the "Install django-auth-ldap" section to specify "pip3" for the command. This had me scratching my head for the past few days because I ran `pip install django-auth-ldap` and ldap was not running when netbox was restarted at the end of the instructions. Using pip3 instead installed the correct module, and ldap finally began working once this was done. 

2. Adding a quip for "you can run `supervisorctl restart netbox` to test your ldap config" to clarify how to troubleshoot would be appreciated. 

3. The following code would be very useful for users to include if they are troubleshooting ldap (configuring logging to /opt/netbox/logs)

```
import logging, logging.handlers
logfile = "/opt/netbox/logs/django-ldap-debug.log"
my_logger = logging.getLogger('django_auth_ldap')
my_logger.setLevel(logging.DEBUG)
handler = logging.handlers.RotatingFileHandler(
   logfile, maxBytes=1024 * 500, backupCount=5)
my_logger.addHandler(handler)
```

**Comment by jeremystretch** [comment]
I've updated the `pip` command in the document since it should have been included during the v2.5 review. I'll leave this open if you want to submit a PR for the rest.

**Comment by till** [comment]
I am trying to use this code snippet above to troubleshoot why LDAP _isn't working_ for us. But all I get is: `Caught LDAPError while authenticating my.user: LDAPError(0, 'Error')` (`my.user` = my actual login).

Is there a way to amp up the logging?
